src/pages/Case.vue

```diff
@@ -0,0 +1,522 @@
+<template>
+  <div>
+    <div>
+      <div>
+        <span>{{ stage }} </span>
+      </div>
+      <button @click="stage++">Increment stage</button>
+    </div>
+
+    <div class="main">
+      <div class="title">New Case</div>
+
+      <div v-if="stage === 0">
+        <div class="col-xs-12 col-md-8 offset-md-2 block border">
+          <div class="wrapper-progressBar">
```

Comment:
change to kebab-case inside the html template

wrapper-progressBar to wrapper-progress-bar